lib/python/alice_check.py

```diff
@@ -0,0 +1,19 @@
+#!/usr/bin/env python3
+import os
+import sys
+
+from sled import Conf
+
+crashed_state_directory = sys.argv[1]
+os.chdir(crashed_state_directory)
+
+dirlist = os.listdir('.')
+
+assert("ALICE.data" in dirlist)
+
+c = Conf()
+c.path(b"ALICE.data")
+
+t = c.tree()
+
+assert(t.get(b"k1") == b"v1")
```

Comment:
this check is run automatically by ALICE when testing for corruption bugs, accepting the test directory as the first argument, which the ALICE framework "corrupted" in some way, to test recovery